lib/preview-web/src/PreviewWeb.tsx

```diff
@@ -307,6 +307,17 @@ export class PreviewWeb<TFramework extends AnyFramework> {
     this.previousSelection = selection;
     this.previousStory = story;
 
+    const { parameters, initialArgs, argTypes, args } = this.storyStore.getStoryContext(story);
```

Comment:
@tmeasday can you explain the move here?

Reply:
Previously we were emitting `STORY_PREPARED` in the `renderStoryToElement` function, which meant it wasn't necessarily emitted on docs pages (if they didn't render the `<Story/>` component).

This way it would only emit for the "root" story in docs. I'm not sure if it should also emit for the other stories rendered on a docs page.

Reply:
SGTM 👍